index.html

```diff
@@ -0,0 +1,332 @@
+    
+                    
+                    <ul class="social-networks">
+                        <li>
+                            <a href="#" class="social-network">
+                                <i class="fa fa-twitter"></i>
+                            </a>
+                        </li>
+                        
+                        <li>
+                            <a href="#" class="social-network">
+                                <i class="fa fa-linkedin"></i>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="social-network">
+                                <i class="fa fa-medium"></i>
+                            </a>
+                        </li>
+                        <li>
+                            <a href="#" class="social-network">
+                                <i class="fa fa-github"></i>
+                            </a>
+                        </li>
+                       <li>
+                        <a href="#" class="social-network">
+                            <i class="fa fa-angellist"></i>
+                        </a>
+                       </li>
+                    </ul>                    
+                </div>
+        
+            </section>
+        </div>
+        <!---------mobile-finish-version--------->
+        <section id="works" class="section-works">
+            <!-----card-works for Tonic------->
+            <div class="card-works tonic">
+                    <img src="./images/SnapshootPortfolio.png" alt="Portfolio Snapshot" class="card-work">
+    
+                <div class="left-block">
+                    <div class="project">
+                        <div class="project-title">                         
+                            <a href="#" class="title">Tonic</a>
+                        </div>
+                        <div class="frame-2">
+                            <a href="#" class="canopy btn-link"> Canopy</a>
+                            <a href="#" class="counter"><img src="./images/Counter.png" alt="Counter image"></a>                
+                            <a href="#" class="back-end-dev btn-link">Back End Dev</a>                            
+                            <a href="#" class="counter"> <img src="./images/Counter.png" alt="counter image"> </a>
+                            <a href="#" class="year btn-link"> 2015 </a>
+                        </div>
+                    </div>
+                    <div class="work-detail">
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+    
+                    <div class="tags">
+                        <span class="tag">html</span>
+                        <span class="tag">css</span>
+                        <span class="tag">javascript</span>
+                    </div>
```

Comment:
- [ ] Please use an unordered list to list out the technologies in all your cards.